x/auth/types/account.go

```diff
@@ -57,7 +57,7 @@ func NewBaseAccountWithAddress(addr sdk.AccAddress) *BaseAccount {
 
 // GetAddress - Implements sdk.AccountI.
 func (acc BaseAccount) GetAddress() sdk.AccAddress {
-	addr, _ := sdk.AccAddressFromBech32(acc.Address)
+	addr := sdk.MustAccAddressFromBech32(acc.Address)
```

Comment:
It diverges from that of the upstream. If this is a minor issue, I think it would be better not to modify it. Any opinions? @zemyblue @tnasu 
This discussion may apply to other lines.

Reply:
As a result of an internal meeting, it was decided to reflect this fix and post the corresponding PR to the upstream as well.